lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -20,9 +20,9 @@
 from courseware.courses import get_course_by_id
 from instructor_analytics.basic import get_proctored_exam_results
 from instructor_analytics.csvs import format_dictlist
-from instructor.views.api import (
-    _do_remote_gradebook,
-    _get_assignment_grade_datatable,
+from instructor.views.instructor_task_helpers import (
+    do_remote_gradebook,
```

Comment:
so you are keeping this file, FYI edX removed it in gingko

Reply:
This file was brought in by the PR that you had done for your remote gradebook changes @amir-qayyum-khan  and I needed to modify it in order to break the circular import.